ui/src/app/components/side-nav.tsx

```diff
@@ -0,0 +1,333 @@
+import {Clickable} from 'app/components/buttons';
+import {ClrIcon} from 'app/components/icons';
+import colors from 'app/styles/colors';
+import {reactStyles} from 'app/utils';
+import {navigate, signInStore} from 'app/utils/navigation';
+import {openZendeskWidget} from 'app/utils/zendesk';
+import {environment} from 'environments/environment';
+import * as React from 'react';
+
+const styles = reactStyles({
+  sideNav: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    justifyContent: 'flex-start',
+    backgroundColor: colors.primary,
+    position: 'absolute',
+    top: '4rem',
+    bottom: '0',
+    zIndex: 1500,
+    flexGrow: 1,
+    width: '10rem',
+    boxShadow: '0px 3px 10px',
+    opacity: 1,
+    transition: 'opacity 0.5s',
+  },
+  sideNavItem: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    width: '100%',
+    margin: 0,
+    paddingLeft: '1rem',
+    textAlign: 'left',
+    textTransform: 'none',
+    height: '2rem',
+    color: colors.white,
+  },
+  sideNavItemActive: {
+    backgroundColor: '#4356a7',
```

Comment:
lets check with Lou we should use variation of color mentioned in colors  and use colorWithWhiteness or add a new color in color file